deploy/k3s/inference_deployment/inference_deployment_template.yaml

```diff
@@ -75,6 +75,13 @@ spec:
           value: placeholder-model-name
         - name: PINAMOD_DIR
           value: /opt/models/pinamod
+        command:
+          [
+            "poetry", "run", "python3", "-m", "uvicorn", "serving.edge_inference_server.fastapi_server:app",
+            "--host", "0.0.0.0",
+            "--port", "8000",
+            "--workers", "32"
+          ]
```

Comment:
This was previously moved to the inference server startup, but it can't be modified for load testing unless it's here. Not sure if we want this merged in though?

Reply:
Its ok to merge this, seems helpful to make it easy to tweak the number of workers.

Although, I dont think we want to run 32 workers by default.